src/resource_format/java_archive.h

```diff
@@ -0,0 +1,13 @@
+#ifndef GODOT_JVM_JAVA_ARCHIVE_H
+#define GODOT_JVM_JAVA_ARCHIVE_H
+
+#include <core/io/resource.h>
+
+class JavaArchive : public Resource {
```

Comment:
Isn't a jar technically a `JvmArchive` and not a `JavaArchive`? I'd rather name it as such

Reply:
JAR stands for Java Archive